.gitlab-ci.yml

```diff
@@ -69,6 +69,34 @@ OpenSUSE:
   <<: *default_config
   <<: *distro_build
 
+ABI_compliance_check:
+  image: fedora:latest
+  stage: deploy
+  variables:
+    # abi-dumper complains if we don't pass -Og
+    CXXFLAGS: "-Og -g3 -gstrict-dwarf"
+    CFLAGS: "-Og -g3 -gstrict-dwarf"
+    # enabling curl is broken on 0.27
```

Comment:
Is it broken when consuming it from a system-package? from conan? both? Would you mind to create a ticket about this? I could check this issue.

Reply:
It's broken when I checkout the tag directly, not from a system package and haven't tried conan.